calico-cloud_versioned_docs/version-3.15/get-started/connect/cc-arch-diagram.mdx

```diff
@@ -0,0 +1,51 @@
+| Image Assurance                | Identifies vulnerabilities in container images that you deploy to Kubernetes clusters.  Includes these components: <br /><br />**Admission controller**<br />Uses Kubernetes Validating Webhook Configuration https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/ to control which images can be used to create pods based on scan results.  <br /><br />**CLI scanner**<br />A CLI binary tool that scans images. Users can upload the results of a scanned image to the backend IA API.  <br /><br />**Pod watcher** <br />Monitors clusters and collects runtime information about which pods are running which images. <br /> | &bull; TCP 8080 to Guardian<br /> &bull;  TCP 6443 to Kubernetes API server |
+|                                | **API and API proxy**<br />Isolates tenant data and ensures that all external access to Image Assurance data is through the API Proxy, which enforces authorization through Kubernetes RBAC. <br /><br />**Note** {{prodname}} does not store registry credentials in its database and does not pull customer images into the {{prodname}} control plane. {{prodname}} uses Google’s encryption at rest feature to encrypt data across the entire datastore. |                                                              |
+| Kubernetes API server          | A Kubernetes component that validates and configures data for the API objects (for example, pods, services, and others). | &bull; TCP 6443 (from all components)                        |
+| kube-controllers               | Monitors the Kubernetes API and performs actions based on cluster state. The {{prodname}} kube-controllers container includes these controllers:<br/><br />&bull; Node<br />&bull; Service<br />&bull; Federated services<br />&bull; Authorization<br />&bull; Elasticsearch configuration<br />&bull; Managed cluster | &bull;  TCP 9094 from Prometheus API service<br />&bull;  TCP 6443 to Kubernetes API server |
+| Prometheus API service         | Collects metrics from {{prodname}} components and makes the metrics available to Manager UI. | &bull;  TCP 6443 to Kubernetes API server<br />&bull;  TCP 9080 to Fluentd<br />&bull; TCP 9900 & 9081 to Prometheus API service |
+| calico-node                    | Bundles key components that are required for networking containers with {{prodname}}:<br/><br />&bull;  Felix<br />&bull;  BIRD<br />&bull;  confd | &bull; TCP 5473 to Typha<br />&bull;  TCP 9900 & 9081 from Prometheus API service |
+| Fluentd                        | Open-source data collector for unified logging. Collects and forwards {{prodname}} logs (flows, DNS, L7) to Elasticsearch. | &bull;  TCP 8080 to Guardian<br />&bull;  TCP 9080 from Prometheus API service |
```

Comment:
I think we should remove `Elasticsearch` and replace it with `log storage`.

Reply:
Fixed.